tools/cli/go-whisk-cli/commands/activation.go

```diff
@@ -31,12 +31,9 @@ import (
     "github.com/spf13/cobra"
 )
 
-const (
-    PollInterval = time.Second * 2
-    Delay        = time.Second * 5
-)
+const MAX_ACTIVATION_LIMIT = 200
```

Comment:
do you intend to keep these limits in the cli if the change is in the backend?
that is, will you hardcode the 200 in this pr or defer this change to a time when you can get the limit from the api info?